websites/M/Merriam-Webster/presence.ts

```diff
@@ -0,0 +1,271 @@
+const presence = new Presence({
+		clientId: "1002734097719894067",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/12lTZIs.png",
```

Comment:
Go to https://discord.com/developers/applications => the application for this presence => rich presence => upload the logo with e.g. the name "logo"
```suggestion
			largeImageKey: "logo",
```

Reply:
No, including the Imgur link is preferable for future maintainability so the author doesn't have to be contacted to change assets.

Reply:
@unicornbetrayal Please revert this

Reply:
> @unicornbetrayal Please revert this

Okay, done

Reply:
> No, including the Imgur link is preferable for future maintainability so the author doesn't have to be contacted to change assets.

Good to know. Will implement this when I get the chance.